libobs/media-io/video-matrices.c

```diff
@@ -272,6 +277,7 @@ bool video_format_get_parameters_for_format(enum video_colorspace color_space,
 		bpc = 12;
 		break;
 	}
+	PRAGMA_WARN_POP
```

Comment:
```c
        uint32_t bpc;
        switch (format) {
        case VIDEO_FORMAT_I010:
        case VIDEO_FORMAT_P010:
        case VIDEO_FORMAT_I210:
                bpc = 10;
                break;
        case VIDEO_FORMAT_I412:
        case VIDEO_FORMAT_YA2L:
                bpc = 12;
                break;
        default:
                bpc = 8;
        }
```

Reply:
`default` is a no go there, it needs to be all cases in the future. But for now, in this PR it's pragma.